src/nft/components/details/detailsV2/DataPageDescription.tsx

```diff
@@ -0,0 +1,41 @@
+import { Trans } from '@lingui/macro'
+import styled from 'styled-components/macro'
+
+import { containerStyles } from './shared'
+import { Tab, TabbedComponent } from './TabbedComponent'
+
+const DescriptionContainer = styled.div`
+  height: 288px;
+  padding: 16px 20px 20px;
```

Comment:
I think the differing bottom padding on this one is a design mistake and they should all be the same with 16px.

Reply:
I think moving `padding: 16px 20px` and setting `padding-left: 0px` and `padding-right: 0px` for the table might be the best. Will talk to design tho